File: 2º Periodo/AED II/Lab/lab01/recursivo.c
#include <stdio.h>
#include <string.h>
void recursao(){
	char s[100];
	int count=0;
	char tag[]={"FIM\n"};
	fgets(s,100,stdin);
	if(strcmp (s,tag)==0) return;
	for(int i=0;s[i]!='\0';i++){
		if(s[i]>='A'&&s[i]<='Z')count++;
	}
	printf("%d\n",count);
	recursao();
}
int main(){
	recursao();
}

File: 2º Periodo/AED II/Lab/lab01/recursivo.java
import java.util.Scanner;
public class recursivo{
	private static Scanner scanner = new Scanner(System.in);
	public static void recursao(){
		String tag = "FIM";
		String s;
		s=scanner.nextLine();
		int count=0;
		if(s.equals(tag)) return;
			for(int i=0;i<s.length();i++){
				char l = s.charAt(i);
				if(l>='A'&&l<='Z')count++;
			}
		System.out.println(count);
		recursao();
	}
	public static void main(String[] args){
		recursao();
		scanner.close();
	}

}
